testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -98,6 +98,23 @@ def test_symlink_file(self, tmpdir):
             assert not fixed.endswith(long_name)
 
 
+class TestOserror(object):
+    u = mda.Universe(MULTIPDB_HOLE)
+
+    errmsg1 = exe_err.format(name='dummy_path', kw='executable')
+
+    errmsg2 = exe_err.format(name='None', kw='executable')
+
+    def test_hole_oserror(self):
+        with pytest.raises(OSError, match=self.errmsg1):
+            hole = hole2.hole(PDB_HOLE, executable = 'dummy_path')
+
+    def test_hole_analysis_oserror(self):
+        with pytest.raises(OSError, match=self.errmsg2):
+            h1 = hole2.HoleAnalysis(self.u, executable=
```

Comment:
Please do address the PEP8 errors as outlined by: https://github.com/MDAnalysis/mdanalysis/pull/3496#issuecomment-1007433066